app/src/test/java/com/android/unio/model/association/AssociationViewModelTest.kt

```diff
@@ -78,6 +88,64 @@ class AssociationViewModelTest {
     Dispatchers.resetMain()
   }
 
+  @Test
+  fun testUpdateFollowIncrement() {
+    `when`(concurrentAssociationUserRepository.updateFollow(any(), any(), any(), any()))
+        .thenAnswer { invocation ->
+          val onSuccess = invocation.arguments[2] as () -> Unit
+          onSuccess()
+        }
+    val association = MockAssociation.createMockAssociation(uid = "1", name = "ACM")
+    val followCount = association.followersCount
+    val user =
+        User(
+            uid = "1",
+            email = "",
+            firstName = "",
+            lastName = "",
+            biography = "",
+            savedEvents = Event.emptyFirestoreReferenceList(),
+            followedAssociations = Association.emptyFirestoreReferenceList(),
+            joinedAssociations = Association.emptyFirestoreReferenceList(),
+            interests = emptyList(),
+            socials = emptyList(),
+            profilePicture = "")
```

Comment:
Are these `association` and `user` objects not extractable (since you use them twice) to the test class? I think each test takes a fresh copy of the object, or am I wrong? Anyway it's not too important but it would be better.

Reply:
Oh, that's right, they execute the @Before before each tests, so we can extract them, yes. Good catch !